src/plugins/intel_cpu/tests/functional/utils/fusing_test_utils.hpp

```diff
@@ -304,6 +306,33 @@ const auto fusingFakeQuantizePerChannel = fusingSpecificParams{std::make_shared<
                 return ov::test::utils::make_fake_quantize(cfg.input, localPrc, 256, newShape);
             }, "FakeQuantize(PerChannel)"}}), {"FakeQuantize"}};
 
+const auto notFusingFakeQuantizePerChannelOrPerTensor = fusingSpecificParams{std::make_shared<postNodesMgr>(std::vector<postNodeBuilder>{
```

Comment:
I would prefer to avoid such kind of randomization in the test.
Lets instead have to different fusingSpecificParams, like "fusingFakeQuantizePerBatch" (with axis 0) and "fusingFakeQuantizeFullTensor" with the const shapes equal to in shape. 
Both types should be parameters of FCNotFuseFQCPUTest